src/main/java/org/broadinstitute/hellbender/tools/spark/pipelines/metrics/GatkReadFilter.java

```diff
@@ -0,0 +1,20 @@
+package org.broadinstitute.hellbender.tools.spark.pipelines.metrics;
+
+import org.apache.spark.api.java.function.Function;
+import org.broadinstitute.hellbender.utils.read.GATKRead;
+
+class GatkReadFilter {
```

Comment:
Once it's in ReadFilterLibrary, you can add a test for it in ReadFilterLibraryUnitTest